frontend/src/app/enums/aspect-metric.enum.ts

```diff
@@ -0,0 +1,12 @@
+import {Unit} from "./unit.enum";
+
+export class AspectMetric {
```

Comment:
I would prefer to locate the unit of PerformanceAspect metrics directly in server side PerformanceAspectType enum. I extracted loading of these aspect types to a global service. Please use loaded Aspect types from that service here and add unit for aspect types to server side enum and dto after it is merged to develop (https://github.com/iteratec/OpenSpeedMonitor/pull/213).